From: Kenichi Handa <handa@m17n.org>
Date: Wed, 12 Jun 2002 00:50:09 +0000
Subject: (miscdic-convert): Force writing the file by iso-2022-7bit.

---
 lisp/international/titdic-cnv.el | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lisp/international/titdic-cnv.el b/lisp/international/titdic-cnv.el
index bcc822c..f309afa 100644
--- a/lisp/international/titdic-cnv.el
+++ b/lisp/international/titdic-cnv.el
@@ -1121,6 +1121,7 @@ the generated Quail package is saved."
       (error "%s does not exist" filename))
   (let ((tail quail-misc-package-ext-info)
 	(default-buffer-file-coding-system 'iso-2022-7bit)
+	(coding-system-for-write 'iso-2022-7bit)
 	slot
 	name title dicfile coding quailfile converter copyright
 	dicbuf)
