From: Glenn Morris <rgm@gnu.org>
Date: Tue, 12 Jun 2007 08:14:14 +0000
Subject: (HAVE_GIF): Doc fix. (LIBGIF): New (already used by Makefile.in).

---
 src/config.in | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/config.in b/src/config.in
index eeb031d..0cfbfd0 100644
--- a/src/config.in
+++ b/src/config.in
@@ -231,7 +231,8 @@ Boston, MA 02110-1301, USA.  */
 /* Define to 1 if you have the `get_current_dir_name' function. */
 #undef HAVE_GET_CURRENT_DIR_NAME
 
-/* Define to 1 if you have the ungif library (-lungif). */
+/* Define to 1 if you have a gif library (default -lungif; otherwise
+   specify with LIBGIF). */
 #undef HAVE_GIF
 
 /* Define to 1 if you have the `grantpt' function. */
@@ -768,6 +769,9 @@ Boston, MA 02110-1301, USA.  */
    Solaris, for example). */
 #undef LD_SWITCH_X_SITE_AUX
 
+/* Define to compiler option to link with gif library, if not -lungif. */
+#undef LIBGIF
+
 /* Define to 1 if localtime caches TZ. */
 #undef LOCALTIME_CACHE
 
