From e68f44846e02443ff5b7dd5370bc372f1ab33485 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ulrich=20M=C3=BCller?= <ulm@gentoo.org>
Date: Fri, 29 Mar 2019 07:47:37 +0100
Subject: Fix compilation with glibc-2.28

Bug-Gentoo: https://bugs.gentoo.org/681904
Origin: https://gitweb.gentoo.org/proj/emacs-patches.git/commit/?id=9f684bdb4781e72cd4c8a94ff40806a8a0a4c6a7
---
 src/s/gnu-linux.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/s/gnu-linux.h b/src/s/gnu-linux.h
index 9863995169d..22f95e4b417 100644
--- a/src/s/gnu-linux.h
+++ b/src/s/gnu-linux.h
@@ -150,7 +150,8 @@ along with GNU Emacs.  If not, see <http://www.gnu.org/licenses/>.  */
 
 #ifdef emacs
 #include <stdio.h>  /* Get the definition of _IO_STDIO_H.  */
-#if defined(_IO_STDIO_H) || defined(_STDIO_USES_IOSTREAM)
+#if defined(_IO_STDIO_H) || defined(_STDIO_USES_IOSTREAM) \
+    || defined _IO_EOF_SEEN || __GNU_LIBRARY__ == 1
 /* new C libio names */
 #define GNU_LIBRARY_PENDING_OUTPUT_COUNT(FILE) \
   ((FILE)->_IO_write_ptr - (FILE)->_IO_write_base)
