From bbcca521125551592f71b9995c5d24221556002b Mon Sep 17 00:00:00 2001
From: Tatsuya Kinoshita <tats@debian.org>
Date: Sat, 21 Jan 2023 22:42:49 +0900
Subject: Avoid loading *.elc under /usr/share/emacs/site-lisp

---
 lisp/startup.el | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/lisp/startup.el b/lisp/startup.el
index cf7f2e39044..8b99cfd95fd 100644
--- a/lisp/startup.el
+++ b/lisp/startup.el
@@ -1077,11 +1077,15 @@ please check its value")
     ;;
 
     ;; Debian startup
-     (if site-run-file
+    (dolist (dir load-path)
+      (if (string-match "^/usr/share/emacs/site-lisp/" dir)
+          (setq load-path (delete dir load-path))))
+    (if site-run-file
         (progn
           ;; Load all the debian package snippets.
           ;; It's in here because we want -q to kill it too.
-          (if (load "debian-startup" t t nil)
+          (if (or (load "/usr/share/emacsen-common/debian-startup.el" t t t)
+                  (load "debian-startup" t t nil))
               (debian-startup debian-emacs-flavor))
           ;; Now the normal site file...
           (load site-run-file t t)))
